.changeset/breezy-lemons-relate.md

```diff
@@ -0,0 +1,16 @@
+---
+"rooks": patch
+---
+
+Remove useTimeTravelState default export
+
+The useTimeTravelState was exported as default and named exports which
+creates possible differences in our dual packaging (cjs, esm). 
+
+This is a potentially breaking change. If you're affected, please raise an issue or
+adapt the import as below.
+
+```typescript
+// Change 'import useTimeTravelState from "rooks"' into
```

Comment:
🤔 I don't think this was the old import. It was probably something like 
`'import useTimeTravelState from "rooks/dist/esm/useTimeTravelState"'` which I don't think anyone used. So this should be fine to remove.